### DISCUSSION TITLE
change port 8080  dependency track for docker container

### DISCUSSION THREAD

**Discussion by bourribab** [DependencyTrack/dependency-track#1343] [General]
Hello,
Is there a way to make Dependency-Track run on a port other than 8080? 
I can't get it to work on another port

thanks for your help

**Comment by stevespringett** [comment]
Of course, ports are defined in the docker-compose.yml

**Reply by bourribab** [reply]
Thank you @stevespringett  for your response. when I use another port or 8080, the frontend is unavailable and the api server container is in healty check :(

**Reply by spmishra121** [reply]
@bourribab,

Please Configure docker-compose like below...
 deploy:
      resources:
        limits:
          memory: 12288m
        reservations:
          memory: 8192m
      restart_policy:
        condition: on-failure
    ports:
      - '8081:8080'
    volumes:
      - 'dependency-track:/data'
    restart: unless-stopped

  dtrack-frontend:
    image: dependencytrack/frontend
    depends_on:
      - dtrack-apiserver
    environment:
      # The base URL of the API server.
      # NOTE:
      #   * This URL must be reachable by the browsers of your users.
      #   * The frontend container itself does NOT communicate with the API server directly, it just serves static files.
      #   * When deploying to dedicated servers, please use the external IP or domain of the API server.
      - API_BASE_URL=http://ip of your server:8081
      # - "OIDC_ISSUER="
      # - "OIDC_CLIENT_ID="
      # - "OIDC_SCOPE="
      # - "OIDC_FLOW="
      # - "OIDC_LOGIN_BUTTON_TEXT="
      # volumes:
      # - "/host/path/to/config.json:/app/static/config.json"
    ports:
      **- "9090:8080"**
    restart: unless-stopped

Note: Changed part is on bold...
